ui/components/component-library/modal-body/modal-body.tsx

```diff
@@ -13,10 +13,6 @@ export const ModalBody: ModalBodyComponent = React.forwardRef(
   ) => (
     <Box
       className={classnames('mm-modal-body', className)}
-      paddingLeft={4}
-      paddingRight={4}
-      paddingBottom={2}
-      paddingTop={2}
```

Comment:
Going have to add individual padding in a subsequent PR because of the prolific nature of the Modal. I'll create another PR which just updates all instances and padding of each individual component to adhere to the designs 

![Screenshot 2023-12-06 at 3 22 21 PM](https://github.com/MetaMask/metamask-extension/assets/8112138/29eda096-2f9d-48ad-80e9-cd45810637d9)